### DISCUSSION TITLE
TimePicker

### DISCUSSION THREAD

**Discussion by Mrking-wst** [HandyOrg/HandyControl#1324] [General]
TimePicker Control , How to binding the selected time value in VM property !

The xaml code like:
     Text="{Binding RemindTime, Converter={StaticResource Stringw2DateTimeConverter}, UpdateSourceTrigger=PropertyChanged}"

it's throw a error.
like:
System.NullReferenceException

  在 HandyControl.Controls.TimePicker.CoerceDisplayTime(DependencyObject d, Object value)...

Whatever that I need to do

**Comment by dthoeni** [comment]
1. Don't set the text. Set the SelectedTime instead.
2. There's a bug in the code. The Control tries to set the DisplayTime to a Clock which isn't instantiated yet.

I have the same problem and am looking for a solution myself.

**Reply by dthoeni** [reply]
I solved it like this:

Forcing the UI to instanciate the Clock BEFORE the SelectedTime is set.

`<StackPanel>
    <StackPanel.Resources>
        <hc:ListClock x:Key="lc"/>    
     </StackPanel.Resources>
    <hc:TimePicker ShowClearButton="True" Clock="{StaticResource lc}" SelectedTime="{x:Static system:DateTime.Now}"/>
</StackPanel>`
